pkg/states/state_manager.go

```diff
@@ -45,6 +45,7 @@ func SetDeactivated(userSignup *v1alpha1.UserSignup, val bool) {
 
 	if val {
 		setState(userSignup, v1alpha1.UserSignupStateApproved, false)
+		setState(userSignup, v1alpha1.UserSignupStateDeactivating, false)
```

Comment:
don't forget to modify also the unit test

Reply:
Done.